extensions/Cake.Issues.DocFx.yml

```diff
@@ -2,18 +2,22 @@ Type: Addin
 Name: Cake.Issues.DocFx
 NuGet: Cake.Issues.DocFx
 Assemblies:
-- "/**/Cake.Issues.DocFx.dll"
+- /**/Cake.Issues.DocFx.dll
 Repository: https://github.com/cake-contrib/Cake.Issues.DocFx.git
 ProjectUrl: https://cakeissues.net/
 Author: BBT Software AG
-Description: |-
+Description: >-
```

Comment:
This is interesting!?!?!

Evidently YamlDotNet decided to use `>-`  rather than `|-` but I am not sure what the nuance is between the two. I hope this will not cause any problem with the documentation generation process.

Reply:
I quick a quick research and turns out `>-` is called "folded block scalar". It's used when long lines are split into shorter lines for readability. [Reference](https://yaml.org/spec/1.2/spec.html#id2796251)

Reply:
Interesting indeed... A quick test with YamlDotNet didn't seem to make any difference.